### DISCUSSION TITLE
Inquiry Regarding Prerequisites for Smart Contract Audits, Security, and DeFi Course

### DISCUSSION THREAD

**Discussion by sathwikNARKEDimilli29** [Cyfrin/security-and-auditing-full-course-s23#46] [Q&A] [answered]
After completing a 32-hour blockchain development tutorial with Hardhat, I'm keen on the Smart Contract Audits, Security, and DeFi Course. Could anyone confirm if the aforementioned tutorial covers the prerequisites for this course? Additionally, I'd like to inquire about how much knowledge in Fundary is required for the same.

**Comment by akhilmanga** [comment]
Do latest Patrick Collins sir's solidity course. Foundry will be covered.

Complete this: https://youtube.com/playlist?list=PL4Rj_WH6yLgWe7TxankiqkrkVKXIwOP42&si=B4NHX-o8iK1zkGg7

**Comment by PatrickAlphaC** [comment, marked as answer]
To be honest, you could probably just jump right into the security course. I'd say, just pay attention in the review/refresher section. We go over some basic foundry use. 

You'll be able to pick it up pretty quick. The spots where it's difficult, just use Google & ChatGPT/Phind to fill the gaps, and also the foundry docs. 

Have fun!
